aspnet/fundamentals/localization/sample/Controllers/AccountController.cs

```diff
@@ -0,0 +1,467 @@
+﻿using System;
```

Comment:
I understand that it's part of the template, but I'm suggesting that we remove the concept of an account completely since it's not central to, or referenced by the doc. And yes, you can't just remove it and move on, you have to remove or modify the places that reference it too (Like ManageController bellow). @danroth27, do you have an opinion? Did we want to leave it in here so that people could use it as an out of the box template kind of thing?